composer.json

```diff
@@ -10,6 +10,14 @@
     ],
     "homepage": "https://github.com/graze/dog-statsd",
     "license": "MIT",
+    "config": {
+        "platform": {
+            "ext-simplexml": "1.0",
```

Comment:
Minor observation: Although `simplexml` is listed as a required extension when this section does not exist, there are actually no complaints when this line is removed. I'm not sure why. If you see the same behaviour, consider simplifying by removing the line.

Reply:
I guess xmlwriter implies simplexml?
who knows.